book/src/ch02-02-functions.md

```diff
@@ -144,7 +150,10 @@ is specified too, as `-> i32`.
 Whereas the last expression in a block implicitly becomes that blocks return
 value, explicit `return` statements always return from the entire function:
 
-```mun
+```mun,ignore
```

Comment:
```suggestion
```mun
```

Reply:
This should compile if you convert `return bar + 3;` into `return 3;`, I think

Reply:
Shouldn't it be a separate pr?